readme.txt

```diff
@@ -0,0 +1,56 @@
+=== Komoju Payments ===
+Tags: WooCommerce,Payment Gateway,Komoju
+Requires at least: 5.3
+Tested up to: 5.4
+Stable tag: trunk
+Requires PHP: 7.2
```

Comment:
For the requirements I just took the WooCommerce 4 requirements, since the code for this plugin is pretty light and WooCommerce is a crucial dependency for us.